randovania/games/samus_returns/exporter/hint_namer.py

```diff
@@ -87,26 +62,14 @@ def format_guaranteed_resource(
             location_name,
         )
 
+    @override
     def format_location_hint(
         self, game: RandovaniaGame, pick_hint: PickupHint, hint: LocationHint, with_color: bool
     ) -> str:
-        msg = self.location_formatters[hint.precision.location].format(
-            game,
-            dataclasses.replace(pick_hint, pickup_name=pick_hint.pickup_name),
-            hint,
-            with_color,
-        )
+        msg = super().format_location_hint(game, pick_hint, hint, with_color)
         return f"{msg}\n"
```

Comment:
I know this isn't something of this branch, but this feels like something that needs to be handled in the patch data factory, not here 🤔